src/commands/moderation/warn/listWarnings.ts

```diff
@@ -17,15 +17,12 @@ const noConnEmbed = makeEmbed({
     color: Colors.Red,
 });
 
-const noPermEmbed = makeEmbed({
-    title: 'Warn',
-    description: 'You do not have permission to use this command.',
-    color: Colors.Red,
-});
-
 export const listWarnings: CommandDefinition = {
     name: ['warnings', 'listwarn', 'listwarnings', 'warns'],
-    requiredPermissions: ['BanMembers'],
+    requirements: {
+        permissions: ['BanMembers'],
```

Comment:
```suggestion
```
Not a comment on the PR, but @benw202 : I'd suggest removing the permissions, it really doesn't have much of a point with the roles there anyway.